resources/assets/js/features/chat/components/UserList/UserListContainer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {getTranslate} from 'react-localize-redux';
+import ChatService from 'app/services/ChatService';
+import {addUsersToList} from '../../actions';
+import UserItem from './UserItem';
+import {ListGroup, ListGroupItem} from 'reactstrap';
+import '../../styles/user-list.scss';
+
+class UserListContainer extends React.Component {
+    constructor() {
+        super();
+    }
+
+    componentWillMount() {
+        this.getUserList();
+    }
+
+    getUserList() {
+        const {addUsersToList} = this.props;
+
+        ChatService.getOthersUser()
```

Comment:
it seems that you have simple request here, such type of request can provide action^ better create action for fetching users that will fill global state